libcrux-ml-dsa/src/ml_dsa_generic.rs

```diff
@@ -26,6 +26,7 @@ pub(crate) mod multiplexing;
 
 /// Generate a key pair.
 #[inline(always)]
+#[allow(unsafe_code)]
```

Comment:
Are these unsafes here in generic still needed? It looks like you pushed that down into the sampling.

Reply:
:+1: You're right, forgot to drop these exceptions.